src/utils/story-helpers.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
```

Comment:
New file :)

To review this file alone, head over to the other PR: https://github.com/primer/react/pull/2044